### ISSUE TITLE
Hybrid - Launch - App is reloaded everytime that it´s backgrounded and opened again.

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#52874]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.65-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** N
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Open the Expensify app.
2. Open any chat.
3. Background the app.
4. Reopen the app.
5. Verify that the app doesn´t load again and that you remain on the opened chat.
6. Navigate to a different section. (For Example: Search)
7. Background the app.
8. Reopen the app.
9. Verify that the app is not loaded again and that you remain on the opened section.

## Expected Result:
App shouldn´t be reloaded everytime that is backgrounded and reloaded and the user should remain on the same chat or section that before it was backgrounded.

## Actual Result:
The app is reloaded everytime that is backgrounded and reopened and the user returns to inbox instead of remaining on the opened chat or section.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [x] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/3d853be2-afcf-4127-8467-bcdccfe4c643

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by cristipaval** [comment]
ok, I could reproduce on 9.0.65-1

**Comment by cristipaval** [comment]
I wonder if this is the [offending  PR](https://github.com/Expensify/Mobile-Expensify/pull/13288), @Julesssss @war-in

**Comment by cristipaval** [comment]
> I wonder if [this is the offending PR](https://github.com/Expensify/Mobile-Expensify/pull/13288), @Julesssss @war-in

if yes, then I think it doesn't make sense to block the NewDot deployment, given that it needs to be fixed in the OldApp repository.

**Comment by cristipaval** [comment]
More context [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1732183180550919).
TLDR: It seems that the root cause is from the OldApp code, so it might not make sense to deploy the App deployment.

**Comment by Julesssss** [comment]
I'd prefer to fix this, it should be as simple as changing the Activity launch type. I will test a build

**Comment by Julesssss** [comment]
PR for fix: https://github.com/Expensify/Mobile-Expensify/pull/13296

**Comment by Julesssss** [comment]
Alright, so we just need an App bump for this to be testable by QA.

**Comment by Julesssss** [comment]
CP'd build in progress https://github.com/Expensify/App/actions/runs/11975270560

**Comment by Julesssss** [comment]
Reopening for QA verification: https://github.com/Expensify/App/issues/52874

**Comment by Julesssss** [comment]
https://expensify.slack.com/archives/C9YU7BX5M/p1732291578881379

**Comment by chiragsalian** [comment]
Fix has been tested - https://expensify.slack.com/archives/C9YU7BX5M/p1732293554743679?thread_ts=1732291578.881379&cid=C9YU7BX5M
removing blocker label.

**Comment by alexpensify** [comment]
Keeping closed - this one was handled internal
